src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/Hop.tsx

```diff
@@ -298,7 +298,13 @@ export const Hop = ({
               <Flex color='text.subtle'>
                 <FaGasPump />
               </Flex>
-              <Amount.Fiat value={networkFeeFiatPrecision ?? '0'} display='inline' />
+              {!networkFeeFiatUserCurrency ? (
+                <Tooltip label={translate('trade.tooltip.continueSwapping')}>
+                  <Text translation={'trade.unknownGas'} fontSize='sm' />
+                </Tooltip>
+              ) : (
+                <Amount.Fiat value={networkFeeFiatUserCurrency} display='inline' />
+              )}
```

Comment:
The only actual diff, all the rest is removing the `fiatPrecision` or `userCurrencyPrecision` which were both incorrect, a user currency is not a crypto amount so no precision here

Reply:
Makes sense ser